src/main/java/Duke.java

```diff
@@ -6,5 +13,49 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can i do for you?\n");
+
+        Scanner input = new Scanner(System.in);
+        ArrayList<String> list = new ArrayList<>();
+        int i = 1;
+
+        while (true){
+            String words = input.nextLine();
+            Task t = new Task(words);
+
+            if (t.description.equals("bye")){
+                System.out.println("Bye! Hope to see you again");
+                break;
+            }
+
+            else if (t.description.equals("list")) {
```

Comment:
Maybe you need to re look at the correct form of if-else statements similar to the coding standards.